docs/updating/from_3.3/update_from_3.3.md

```diff
@@ -235,6 +235,34 @@ Run the following scripts:
     psql <database_name> < vendor/ibexa/installer/upgrade/db/postgresql/ibexa-3.3.8-to-3.3.9.sql
     ```
 
+### v3.3.14
+
+#### Update VCL
```

Comment:
```suggestion
#### VCL configuration for Varnish
```

Reply:
VCL stands for "Varnish Configuration Language", so what would you say to possibly "Update VCL configuration"?

Reply:
I know, I thought it would look elegant if we brought it to synch with the header from line 40: "VCL configuration for Fastly"